app/views/shared/_bar.html.haml

```diff
@@ -9,22 +9,23 @@
             = link_to bar.name, bar_path(bar)
           %p.card-text
             Rating:
-            #{bar.rating}
+            #{ bar.rating }
           %p.card-text
             Address:
-            #{bar.address}
+            #{ bar.address }
           %p.card-text
             = truncate(bar.description, length: 100)
           %small
             created by:
-            #{bar.user.username if bar.user}
+            #{ bar.username if bar.user }
```

Comment:
String interpolation is an exception from empty spaces rule ;). But anyways, consider using 
```
= bar.rating
= bar.address
= bar.username 
```
Also, is it possible for bar to be added without user?  Do we need that condition?

Reply:
fixed